[2025-01-09T21:05:11.664+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: export_import_data.insert_data manual__2025-01-09T21:05:04.555973+00:00 [queued]>
[2025-01-09T21:05:11.675+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: export_import_data.insert_data manual__2025-01-09T21:05:04.555973+00:00 [queued]>
[2025-01-09T21:05:11.676+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-09T21:05:11.698+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): insert_data> on 2025-01-09 21:05:04.555973+00:00
[2025-01-09T21:05:11.705+0000] {standard_task_runner.py:60} INFO - Started process 3453 to run task
[2025-01-09T21:05:11.709+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'export_import_data', 'insert_data', 'manual__2025-01-09T21:05:04.555973+00:00', '--job-id', '2048', '--raw', '--subdir', 'DAGS_FOLDER/export_import_data.py', '--cfg-path', '/tmp/tmp3ncxn9ou']
[2025-01-09T21:05:11.712+0000] {standard_task_runner.py:88} INFO - Job 2048: Subtask insert_data
[2025-01-09T21:05:11.983+0000] {task_command.py:423} INFO - Running <TaskInstance: export_import_data.insert_data manual__2025-01-09T21:05:04.555973+00:00 [running]> on host 803f979fc22a
[2025-01-09T21:05:12.085+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='avbershits' AIRFLOW_CTX_DAG_ID='export_import_data' AIRFLOW_CTX_TASK_ID='insert_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-09T21:05:04.555973+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-09T21:05:04.555973+00:00'
[2025-01-09T21:05:12.104+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-09T21:05:12.164+0000] {base.py:83} INFO - Using connection ID 'postgres_db' for task execution.
[2025-01-09T21:05:12.189+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-09T21:05:12.202+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=export_import_data, task_id=insert_data, execution_date=20250109T210504, start_date=20250109T210511, end_date=20250109T210512
[2025-01-09T21:05:12.259+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-09T21:05:12.494+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
